PlayRho/Collision/Shapes/ChainShapeConf.hpp

```diff
@@ -115,14 +115,14 @@ class ChainShapeConf : public ShapeBuilder<ChainShapeConf>
     /// @brief Gets a vertex by index.
     Length2 GetVertex(ChildCounter index) const
     {
-        assert((0 <= index) && (index < GetVertexCount()));
+        assert(index < GetVertexCount());
```

Comment:
Why am I not seeing a warning on my clang builds with the original line here? I mean from recollection, `ChildCounter` is unsigned so `index` is always greater-than-or-equal-to zero which I thought `-Wall` -Wextra` warns about. I need to investigate this.

Reply:
Idk why your not seeing a warning... But yes that's why I proposed these changes.